packages/nx/src/native/tasks/hashers/hash_project_files.rs

```diff
@@ -18,6 +18,7 @@ pub fn hash_project_files(
     let mut hasher = xxhash_rust::xxh3::Xxh3::new();
     for file in collected_files {
         hasher.update(file.hash.as_bytes());
+        hasher.update(file.file.as_bytes());
```

Comment:
This looks good to me, can you also update hash_workspace_files? I imagine it has the same issue.

Reply:
Updated.

Reply:
@AgentEnder hmm, CI failed but I don't see why. Might need some help on that one.